nl/guides/for-students/README.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Van start met Dodona voor studenten
+description: "Tutorial: van start met Dodona voor studenten"
+next: ./login-and-settings/
+---
+
+# Dodona voor studenten
+
+Als student kan je [registreren](./courses/#registreren-voor-een-cursus) voor één of meer [cursussen](). Elke cursus bevat een reeks [oefeningen](./exercises/#navigeren-naar-een-oefening) die gegroepeerd zijn in [oefeningenreeksen](). Voor elke oefening kan je herhaald [oplossingen](./exercises/#navigeren-naar-een-oplossing) [indienen](./exercises/#indienen-van-een-oplossing) die automatisch worden beoordeeld en van [feedback](./exercises/#interpreteren-van-feedback) voorzien. Deze acties vereisen echter dat je over een [gebruikersaccount](./login-and-settings/#aanmelden) beschikt en dat je [aangemeld](./login-and-settings/#aanmelden) bent op Dodona.
+
+## Nederlandse handleidingen voor studenten
+
+* [Aanmelden & gebruikersinstellingen](./login-and-settings/)
+* [Oefeningen handleiding](./exercises/)
+* [Cursus handleiding](./courses/)
+
+## Contact opnemen
+
+Als je met vragen zit over de werking van Dodona of als er interessante ideeën bij je opborrelen over hoe we het platform zouden kunnen verbeteren of uitbreiden, neem dan gerust contact met ons op. Alle commentaren en suggesties zijn meer dan welkom. Navigeer daarvoor naar de **contactpagina** door in de **voettekst** (onderaan elke pagina) op <span class="guilabel">Contact</span> te drukken.
```

Comment:
The bold text here is useless, so remove it, guilabel can probably be  replaced by backticks like mentioned before. Contactpagina should be a link to the actual contactpage.